app/views/form.html

```diff
@@ -24,10 +24,12 @@
   <label class="col-sm-2 control-label">Sted</label>
   <div class="col-sm-10">
     <select name="Place" id="mySelect"
+            ng-change="updateText()"
             required
             ng-options="venue as venue.name for venue in venues"
             ng-model="event.venue"><option value = "">Velg sted</option></select>
- </div>
+    <p ng-show="isEdit">Du har valgt: {{ venuename }}</p>
```

Comment:
Why do you need `ng-change` and `venuename`? Why not use `event.venue.name` directly?